packages/malloy/src/lang/test/test-translator.ts

```diff
@@ -182,55 +187,62 @@ export class TestTranslator extends MalloyTranslator {
    *     query: aturtle is { group_by: astr; aggregate: acount }
    *   }
    */
-  internalModel: ModelDef = {
```

Comment:
(This is now set in the constructor, since it too is parameterized on `dialect`.)